atlas/additional-guides/build-notifications.mdx

```diff
@@ -0,0 +1,18 @@
+There's a number of ways you can get notified about finished Atlas builds
+
+## E-mail notifications based on GitHub comments
+Atlas posts comments on commit pages once a build is complete. These comments contain information about status of the build as well as link to the build logs and to the Atlas environment.
```

Comment:
Just a few word structure changes to this sentence:

_These comments contain information about the status of the build as well as a link to the build logs and to the Atlas environment._